homeassistant/components/switchbot_cloud/sensor.py

```diff
@@ -89,6 +90,13 @@
     native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
 )
 
+POWER_CONSUMPTION_DESCRIPTION_IN_MA = SensorEntityDescription(
+    key="electricityOfDay",
```

Comment:
Consider  using a constant for the key like the other descriptions do - for code consistency.

Reply:
ok